ramp-engine/ramp_engine/base.py

```diff
@@ -1,18 +1,24 @@
 import logging
+from logging.handlers import RotatingFileHandler
+import os
 from abc import ABCMeta, abstractmethod
 
+log_dir = "logs"
```

Comment:
Can't we make this configurable somehow? 
The issue is that this log folder will depend on the location from which we run a script as far as I understand (an also happens at import). It might be better to have a function that is called as necessary that takes as input a config object and returns this logger. Not sure how easy that would be though.